README.md

```diff
@@ -36,13 +36,13 @@ Here follows a table to ease migrating Travis build that were using config offer
 | .travis.helm_install.yml                  | Preinstalled                                                                |
 | .travis.home_bin_path.yml                 | Not yet determined                                                          |
 | .travis.java.yml                          | Not yet determined                                                          |
-| .travis.java_config.yml                   | Not yet determined                                                          |
+| .travis.java_config.yml                   | [actions/setup-java](#actionssetup-java)                                    |
 | .travis.java_docker.yml                   | Not yet determined                                                          |
 | .travis.jq_install.yml                    | Preinstalled                                                                |
 | .travis.kcadm_install.yml                 | Not yet determined                                                          |
 | .travis.kubepug_install.yml               | [setup-kubepug](.github/actions/setup-kubepug/action.yml)                   |
 | .travis.kubernetes_install.yml            | Preinstalled                                                                |
-| .travis.maven_config.yml                  | Not yet determined                                                          |
+| .travis.maven_config.yml                  | [s4u/maven-settings-action](#s4umaven-settings-action)                      |
```

Comment:
I realized that I left the community action in there, please tell if I should change it

Reply:
right, I would probably point here to the actions/setup-java section and merge the two section together

you can keep the final line pointing to s4u/maven-settings-action that maybe can be useful for more complex use cases

Reply:
Ok will do, thanks :+1: